doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+===================================== =============================================
+Pan                                   :kbd:`Alt`-click and drag
+Zoom                                  :kbd:`Alt`-right-click and drag
+"                                     Mouse wheel up/down
+Frame selected nodes                  :kbd:`F`
+Enter `Box` node (subgraph)	          :kbd:`↓`
+Leave `Box` node (subgraph)           :kbd:`↑`
+Search for nodes                      :kbd:`Ctrl`-:kbd:`F`
+Bookmark node                         Right-click on node
+===================================== =============================================
+```
+
+
+## _Viewer_ Tab ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover inside the _Viewer_ tab.
+
+
+### 2D Images ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut/gesture
+===================================== =============================================
+Pan                                   :kbd:`Alt`-click and drag
+Zoom                                  :kbd:`Alt`-right-click and drag
+"                                     Mouse wheel up/down
+Frame image                           :kbd:`F`
+Center image at 1:1 scale             :kbd:`Home`
+Toggle red channel                    :kbd:`R`
+Toggle green channel                  :kbd:`G`
+Toggle blue channel                   :kbd:`B`
+Toggle alpha channel                  :kbd:`A`
+===================================== =============================================
+```
+
+### 3D Scenes ###
+
+```eval_rst
+========================================== =====================================
+Action			                           Shortcut/gesture
+========================================== =====================================
+Tumble                                     :kbd:`Alt`-click and drag
+Track                                      :kbd:`Alt`-middle-click and drag
+Dolly                                      :kbd:`Alt`-right-click and drag
+"                                          Mouse wheel up/down
+Expand selection                           :kbd:`↓`
+Fully expand selection                     :kbd:`Shift`-:kbd:`↓`
+Collapse selection                         :kbd:`↑`
+Frame selection                            :kbd:`F`
+Fit clipping planes to selection           :kbd:`Ctrl`-:kbd:`K`
+Frame and fit clipping planes to selection :kbd:`Ctrl`-:kbd:`F`
+========================================== =====================================
+```
+
+
+## _Script Editor_ Tab ##
+
+### Text Entry ###
+
+```eval_rst
+===================================== =============================================
+Action                                Shortcut/gesture
+===================================== =============================================
+Pull node into script                 Middle-click and drag node in *Node Graph* tab
```

Comment:
I think the new actions in this section are as confusing than the originals (which were themselves not great). We're not pulling or inserting anything, and we're certainly not modifying the script itself. We're referring to existing nodes/plugs/values. The technical event occuring is a drop event, so maybe we can just call this "Drop node into editor" and so on?

Reply:
I think the new descriptions are overly verbose as well... I have to scroll just to read them and most of the copy is redundant for each row. Maybe we can simplify them using a handy Note for this subsection saying every action here refers to dropping onto the ScriptEditor's text input field?

Reply:
"Drop" works. I'll see about the note.